target/scripts/startup/setup-stack.sh

```diff
@@ -426,7 +426,7 @@ function _setup_ldap
   if [[ -f /etc/postfix/ldap-aliases.cf ]] && [[ -f /etc/postfix/ldap-groups.cf ]]
   then
     postconf -e "virtual_alias_maps = ldap:/etc/postfix/ldap-aliases.cf, ldap:/etc/postfix/ldap-groups.cf" || \
-    _notify 'inf' "==> Warning: /etc/postfix/ldap-aliases.cf or /etc/postfix/ldap-groups.cf not found"
+    _notify 'war' "/etc/postfix/ldap-aliases.cf or /etc/postfix/ldap-groups.cf not found"
```

Comment:
I am confused. First there is a check, if the two files exist. But if `postconf` fails, a warning that one of those files is missing is printed 😮

Reply:
Haha, I just noticed that it is weird that `notify 'inf' 'warning:'` was used, so I changed it to `notify 'war'` instead without looking what is being checked at all 😂 I think this line needs to go into an `else` statement...